# TransformationMatrix.msg
std_msgs/MultiArrayLayout layout
float64[16] matrix